test/test-platform-output.c

```diff
@@ -236,5 +236,28 @@ TEST_IMPL(platform_output) {
   printf("  version: %s\n", uname.version);
   printf("  machine: %s\n", uname.machine);
 
+  /* Win and Darwin won't define RUSAGE_THREAD */
+#if !defined(_WIN32) && !defined(__APPLE__) && !defined(RUSAGE_THREAD)
```

Comment:
They support it now, right?

Reply:
Yes, but not thru' `RUSAGE_THREAD` as pointed here https://github.com/libuv/libuv/pull/4666/files#diff-73228ae445cdc20e1147c8db069be36a945fcbc020b6ae92b8d9d46787c80a1eR1088

Reply:
Oh, right. I think it'd be better to explicitly list the platforms where we expect to see ENOTSUP (none? Linux and the BSDs all support RUSAGE_THREAD; maybe not Solaris but who cares?)

Reply:
Honestly, idk... for now, we just know that Windows, macOS, Linux and FreeBSD does support them.

For now, we can say that macOS, windows and any kernel with `RUSAGE_THREAD` can access this feature. Maybe update the comment to reflect that instead?

Reply:
I think it should be fine to remove the ifdef guards. All the tier 1 and 2 platforms support with maybe the exception of Solaris-like platforms, and we can just wait for the bug reports to come in for the latter.

I don't like ifdef'ing on platform constants in tests because experience teaches they have a habit of hiding bugs.

Reply:
Let me try that, maybe CI won't cry.